app/src/main/java/com/ntg/user/mvpsample/task_details/TaskDetailFragment.java

```diff
@@ -0,0 +1,30 @@
+package com.ntg.user.mvpsample.task_details;
+
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.ntg.user.mvpsample.R;
+
+/**
+ * A simple {@link Fragment} subclass.
+ */
+public class TaskDetailFragment extends Fragment {
```

Comment:
no need for constructor in fragment if you won't pass any parameters.
remove comments inside method , method name / divination should be clear enough to understand logic.